Makefile

```diff
@@ -61,10 +61,15 @@ debug: base
 .PHONY: test
 test: lint
 	BCRYPT=no npx mocha --recursive --exit
+
 .PHONY: test-full
 test-full: lint
 	npx mocha --recursive --exit
 
+.PHONY: test-fast
+test-fast:
```

Comment:
Maybe this should run `node_version` like `test-integration` and `test-unit`? Or it could run `lint` like `test` and `test-full`.

Reply:
I tend to fix lint at the very end that's why I didn't add `lint` or `node_version`.

Reply:
I also like linting and testing separately. :+1: Note that `node_version` doesn't run `lint`, it just runs `npm install` and checks the Node version.